parsing/dialects/intermediate/dialects/dialect1/config.fmpp

```diff
@@ -77,6 +77,7 @@ data: {
       "org.apache.calcite.sql.SqlCreateTableDialect1",
       "org.apache.calcite.sql.SqlDateTimeAtLocal",
       "org.apache.calcite.sql.SqlDateTimeAtTimeZone",
+      "org.apache.calcite.sql.SqlDeallocatePrepare",
```

Comment:
nit: Put in alphabetical order (e.g. after SqlDdlNodes). Also, can you remove the duplicate SqlDdlNodes while you're here?

Reply:
Yeah I'll update that